example-metrics/pom.xml

```diff
@@ -33,7 +33,7 @@
         </dependency>
         <dependency>
             <groupId>org.specs2</groupId>
-            <artifactId>specs2-junit_2.11</artifactId>
+            <artifactId>specs2-junit_2.12</artifactId>
```

Comment:
You need to parametrize these values so we can cross build to 2.12 and 2.13
Example: https://github.com/oracle/wookiee/blob/master/wookiee-core/pom.xml#L33